node_monitor/node_monitor.py

```diff
@@ -125,12 +125,12 @@ def broadcaster(node_provider_id: str,
                 if self.slack_bot:
                     channels = slack_channels.get(node_provider_id, [])
                     if channels:
-                        err1 = self.slack_bot.send_messages(channels, message)
+                        err1 = self.slack_bot.send_messages(channels, subject, message)
             if preferences['notify_telegram'] == True:
                 if self.telegram_bot:
                     chats = telegram_chats.get(node_provider_id, [])
                     if chats:
-                        err2 = self.telegram_bot.send_messages(chats, message)
+                        err2 = self.telegram_bot.send_messages(chats, subject, message)
             return None
```

Comment:
Let's keep the send_messages() api the same as before, and instead concatenate the subject+message here, before the function gets called.